tsfile/src/main/java/org/apache/iotdb/tsfile/common/conf/TSFileDescriptor.java

```diff
@@ -105,39 +101,48 @@ private void loadProps() {
     Properties properties = new Properties();
     try {
       properties.load(inputStream);
-      conf.groupSizeInByte = Integer
-          .parseInt(properties.getProperty("group_size_in_byte", conf.groupSizeInByte + ""));
-      conf.pageSizeInByte = Integer
-          .parseInt(properties.getProperty("page_size_in_byte", conf.pageSizeInByte + ""));
-      conf.maxNumberOfPointsInPage = Integer.parseInt(
+      TSFileConfig.groupSizeInByte = Integer
+          .parseInt(
+              properties.getProperty("group_size_in_byte", TSFileConfig.groupSizeInByte + ""));

```

Comment:
Use methods like Integer.toString(), Long.toString() instead of string concatenation.

Reply:
ok